### DISCUSSION TITLE
dagster-airbyte and having to use the CLI to apply changes

### DISCUSSION THREAD

**Discussion by kjoconnor** [dagster-io/dagster#13782] [General]
I love being able to [configure Airbyte connections in Python](https://docs.dagster.io/guides/dagster/airbyte-ingestion-as-code), but what's the reasoning behind having to then drop out to a CLI to actually apply the changes? Is it just because there's no sane place for that functionality to love in Dagster currently?

Thanks!

**Comment by sryza** [comment]
The thinking is that we think of applying the changes as a deploy step, analogous to uploading a container to a container registry. Dagster generally doesn't take responsibility for CI/CD, so the CLI is expected to be used as part of a Github Action or similar.

**Reply by kjoconnor** [reply]
Got it. Thank you!
